core/build.gradle

```diff
@@ -112,6 +112,7 @@ configurations {
 
 dependencies {
     implementation project(':rpc')
+    implementation group: 'org.apache.hbase', name: 'hbase-common', version: '3.0.0-alpha-1'
```

Comment:
It shouldn't depend on hbase.
For this kind of design thing, you should have asked Suzuki-san before start implementing.

Reply:
We did discuss this before already. I tried to take only part of encoding from hbase but the code relating to many files so to make it simple I add hbase-common ad dependency. @brfrn169 how do you think?

Reply:
There is actually a huge misunderstanding because Toshi (and myself) didn't expect to depend on `hbase-common` ...
So, for this kind of thing like adding a big dependency, it should be asked before actually implementing otherwise there would be a lot of code rewrites.

Reply:
I don't think we should have the `hbase-common` dependency in Scalar DB. `hbase-common` doesn't assume it's used outside of HBase, and it has a lot of unrelated code for Scalar DB.

> We did discuss this before already.

I thought I said we could copy the necessary part of the HBase code, right? We might be able to have a license issue if we completely copy it, though.

Reply:
Yes, in the beginning, I tried to copy the necessary part but turned out many files and other dependency packages as well, so for making it simple I change to add `hbase-common`.  I haven't thought that we shouldn't add this dependency.